interface/wx/filename.h

```diff
@@ -1246,6 +1246,18 @@ class wxFileName
     */
     bool SetPermissions(int permissions);
 
+    /**
+        Converts URL into a well-formed filename.
+        The URL must use the @c file protocol.
```

Comment:
Documentation should mention what happens if this is not the case.

Also missing `@since 3.1.3`.

Reply:
@vadz ,
The original didn't mention that.
I will look into fixing it.